source/_includes/common-tasks/backups.md

```diff
@@ -99,6 +99,8 @@ This creates a backup instantly. You can create a manual backup at any time, irr
 
 ### Downloading your local backups
 
+When downloading the backup from the backup page in Home Assistant, the backup is decrypted on the fly so that you then can view the data using your favorite archive tool. This is done for all backup locations, also when you download from Home Assistant Cloud.
```

Comment:
```suggestion
When downloading the backup from the Home Assistant backup page, it is decrypted on the fly so that you can view the data using your favorite archive tool. This is done for all backup locations and also when you download from Home Assistant Cloud.
```